hermeznetwork/wallet-ui
# Support custom Wallet Connect Bridge

Closes #623 

### What does this PR does?

We now use a custom Bridge for Wallet Connect. This increases the stability of the connections. I've also added the correct secret on GH.

### How to test?

Set the `REACT_APP_WALLETCONNECT_BRIDGE` environment variable to the correct value and it should now send transactions and signatures fine.

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:
